.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+    - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
do we think and isort and black should be sufficient for python packages? 

I think we dont need prettier ?

Reply:
Probably unnecessary with black formatter already. I'll remove for now. We can always add it back in if need be.